backend/src/main/twirl/MainIOHttp4s.scala.txt

```diff
@@ -0,0 +1,25 @@
+@(groupId: String)
+package @groupId
+
+import cats.effect.{ExitCode, IO, IOApp}
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.server.Router
+import sttp.tapir.server.http4s.Http4sServerInterpreter
+
+import scala.io.StdIn
+
+object Main extends IOApp {
+
+  override def run(args: List[String]): IO[ExitCode] = {
+    implicit val ec = scala.concurrent.ExecutionContext.global
+    val routes = Http4sServerInterpreter[IO]().toRoutes(List(ApiDefinitions.helloServerEndpoint))
+
+    BlazeServerBuilder[IO]
+      .withExecutionContext(ec)
```

Comment:
a to potrzebne nadpisywac? 🤔

Reply:
Thanks ! There is no sense for doing it.